### PR TITLE
[UXIT-1797] Show subcategory tag label on Ecosystem Project Page

### PR DESCRIPTION
**Pull request by barbaraperic**
## 📝 Description

Cards on the Ecosystem Project Page now also include the subcategory tag.

## 📸 Screenshots

Screenshot 2024-11-28 at 09 15 09
Screenshot 2024-11-28 at 09 14 51

### CODE REVIEW

**Comment by CharlyMartin** [comment]
Aside from my comments, I wonder what we should do about this?

Label texts should probably stay on the same line, but the label itself can go to the next line if necessary?

CleanShot 2024-11-28 at 15 46 34@2x

CleanShot 2024-11-28 at 15 45 57@2x

File: src/app/ecosystem-explorer/utils/getEcosystemTagLabels.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+} from '@/constants/paths'
+
+import { getCategoryDataFromDirectory } from '@/utils/categoryUtils'
+
+import type { Category, Subcategory } from '../schemas/CategorySchemas'
+
+type GetEcosystemTagLabelsProps = {
+  category: Category
+  subcategories?: Array<Subcategory>
+}
```
[the remark this answers is not in the document]
Comment (barbaraperic) [reply src/app/ecosystem-explorer/utils/getEcosystemTagLabels.ts:13]: works for me!

File: src/app/ecosystem-explorer/schemas/CategorySchemas.ts [hunk 2]
```diff
@@ -39,3 +39,6 @@ export const DirectorySubcategorySchema = z.object({
   name: z.string(),
   parent_category: CategorySlug,
 })
+
+export type Category = z.infer<typeof CategorySchema>
+export type Subcategory = z.infer<typeof SubcategorySchema>
```
[the remark this answers is not in the document]
Comment (barbaraperic) [reply src/app/ecosystem-explorer/schemas/CategorySchemas.ts:44]: At the glance of it, it seems the `createEnumSchema` returns a string value, I assume if the array has one element?

Screenshot 2024-11-28 at 12 39 18

**Comment by barbaraperic** [comment]
> Aside from my comments, I wonder what we should do about long label names.
> 
> Label texts should probably stay on the same line, but the label itself can go to the next line if necessary.
> 
> What do you think?
> 
> CleanShot 2024-11-28 at 15 46 34@2x
> 
> CleanShot 2024-11-28 at 15 45 57@2x

Yes, I'll check with Filipa :)

**Comment by mirhamasala** [comment]
@barbaraperic This is not correct. The Cards and the pages ONLY show the subcategory tag. There's a Slack thread in the ticket.

**Review by mirhamasala — APPROVED** [review]
@barbaraperic @CharlyMartin - This implementation wasn't correct: We ONLY want to show the subcategory. I pushed a CR. It's ready to go. We have other tickets open to clean up the data. 🙏🏼

**Comment by barbaraperic** [comment]
> @barbaraperic @CharlyMartin - This implementation wasn't correct: We ONLY want to show the subcategory. I pushed a CR. It's ready to go. We have other tickets open to clean up the data. 🙏🏼

Should we have the same border color for the tags? Currently it is white on` /ecosystem-explorer` and blue on individual pages?

**Comment by mirhamasala** [comment]
> Should we have the same border color for the tags? Currently it is white on /ecosystem-explorer and blue on individual pages?


It's how it is on /blog as well so I kept it.

**Pull request merged**